src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Security/SignInControllerTests.cs

```diff
@@ -272,13 +272,19 @@ public async Task ExpiredInvites(int daysOld, bool expectExpiredInviteResult)
                     var globalRoles = GetGlobalRoles();
                     context.Roles.AddRange(globalRoles);
 
+                    var createdDate = DateTime.Now.AddDays(-daysOld);
+                    if (TimeZoneInfo.Local.IsDaylightSavingTime(DateTime.Now))
+                    {
+                        createdDate = createdDate.AddHours(-1);
+                    }
+
                     // Add a global role invite for the user and back-date its created date.
                     context.UserInvites.Add(new UserInvite
                     {
                         // Change the case of the email address in the invite to test this scenario.
                         Email = email.ToLower(),
                         Role = globalRoles.First(),
-                        Created = DateTime.Now.AddDays(-daysOld),
```

Comment:
Would this have been failing if it was DataTime.UtcNow?

Reply:
Yeah I've changed them all to UtcNow now - before I saw your comment :)